ETL/Pipeline/conftest.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+
+@pytest.fixture(name="fake_headers")
+def fixture_fake_headers() -> dict:
+    return {'User-Agent': 'test'}
+
+
+@pytest.fixture(name="fake_product_data")
+def fixture_fake_product_data() -> dict:
+    return {
+        "product_id": 12345,
+        "previous_price": 100,
+        "url": "www.asos.com/shoes",
+        "product_name": "adidas Originals Samba LT trainers"}
+
+
+@pytest.fixture(name="fake_product_list")
+def fixture_fake_product_list() -> list[dict]:
+    return [{
+        'product_id': 205759745,
+        'current_price': '',
+        'previous_price': 85,
+        'url': "https://www.asos.com/adidas-originals/adidas-originals-gazelle-trainers-in-white-and-blue/prd/205759745#ctaref-we%20recommend%20carousel_11&featureref1-we%20recommend%20pers",
```

Comment:
```suggestion
        'url': "https://www.asos.com/adidas-originals/\
        adidas-originals-gazelle-trainers-in-white-and-\
        blue/prd/205759745#ctaref-we%20recommend\
        %20carousel_11&featureref1-we%20recommend%20pers",
```
Adding \ will allow you to use multiple lines and stop pylint from barking

Reply:
Very helpful, thank you